hack/make-rules/test.sh

```diff
@@ -186,7 +186,7 @@ junitFilenamePrefix() {
   # barely fits there and in coverage mode test names are
   # appended to generated file names, easily exceeding
   # 255 chars in length. So let's just sha1sum it.
```

Comment:
I'm guessing that the length and number of APIs increased so that it now exceeds 255 characters